docs/developer.md

```diff
@@ -253,3 +253,45 @@ cobra add commandName
 ```
 
 You should see a new file appear under `./cmd/armadactl/cmd` with the name you specified in the command.
+
+### Setting up OIDC for developers.
+
+Setting up OIDC can be an art.  We found that [Okta Developer Program](https://developer.okta.com/signup/) provides a nice quick-and-dirty way to test OAuth flow.
```

Comment:
I'd suggest removing the "quick-and-dirty" adjective - it seems slightly derisive, and this whole solution seems perfectly acceptable to me.